src/libs/PolicyUtils.js

```diff
@@ -48,7 +48,7 @@ function hasCustomUnitsError(policy) {
  */
 function getPolicyBrickRoadIndicatorStatus(policy, policyMembers) {
     const policyMemberList = lodashGet(policyMembers, `${ONYXKEYS.COLLECTION.POLICY_MEMBER_LIST}${policy.id}`, {});
-    if (hasPolicyMemberError(policyMemberList) || hasPolicyError(policy) || hasCustomUnitsError(policy)) {
+    if (hasPolicyMemberError(policyMemberList) || hasCustomUnitsError(policy)) {
```

Comment:
This is to remove the BrickRoad indicator status from the workspace line if the error is in the workspace itself, otherwise we get a double red dot: 


<img width="359" alt="Screen Shot 2022-09-02 at 5 48 17 PM" src="https://user-images.githubusercontent.com/87341702/188194357-dd635ed8-2947-4c58-8f57-640fd438ed40.png">